packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -152,7 +159,7 @@ export class ChainService implements ChainServiceInterface {
   }
 
   async fundChannel(arg: FundChannelArg): Promise<providers.TransactionResponse> {
-    this.logger.info({...arg}, 'Attempting to fund channel');
+    this.logger.info({...arg}, 'ChainService.fundChannel');
```

Comment:
nit: I don't think we need to include `ChainService` in the string since the `logger` has the module set to `ChainService` already: `logger.child({module: 'ChainService'})`.

Reply:
Good call ce7244d5e